tasks/eva_2369/export_mongo_dbs.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import sys
+
+from ebi_eva_common_pyutils.logger import logging_config
+from ebi_eva_common_pyutils.mongodb import MongoDatabase
+
+logger = logging_config.get_logger(__name__)
+logging_config.add_stdout_handler()
+
+
+def dump_data_from_source(mongo_source: MongoDatabase, top_level_dump_dir):
+    try:
+        logger.info("Running mongodump from source...")
+
+        # Force table scan is performant for many workloads avoids cursor timeout issues
+        # See https://jira.mongodb.org/browse/TOOLS-845?focusedCommentId=988298&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-988298
+        mongo_source.dump_data(dump_dir=os.path.join(top_level_dump_dir, mongo_source.db_name),
+                               mongodump_args={"gzip": "", "forceTableScan": "", "numParallelCollections": "1"})
```

Comment:
I was thinking about this a bit more. You can also add the [--archive option with a target file name](https://stackoverflow.com/a/63720355) so that a single file is created per database instead of multiple files. Since we don't intend to restore this data later on, archiving the data this way looks like a better option.

Reply:
updated the code to have archive option. Since --archive cannot be used with --out, have created a separate method in eva-common-pyutils for the archive function. Have raised a PR for that change also.